modules/nf-core/sratools/fasterqdump/main.nf

```diff
@@ -22,11 +22,6 @@ process SRATOOLS_FASTERQDUMP {
     def args = task.ext.args ?: ''
     def args2 = task.ext.args2 ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
-
-    // WARNING: Paired-end data extracted by fasterq-dump (--split-3 the default)
-    // always creates *_1.fastq *_2.fastq files but sometimes also
-    // an additional *.fastq file for unpaired reads which we ignore here.
```

Comment:
I get the reason for the widened regex on the output, but are you not still worried about what this comment says in the case of non-10X data, i.e. that you'll pick up the unpaired reads

Reply:
Does there need to be a '10X' flag somewhere to allow the new behaviour in a backwards-compatible manner?

Reply:
Actually, we needed to make very little changes to accomodate the widened regex in this PR:
https://github.com/nf-core/fetchngs/pull/146

Reply:
Fair enough, just wanted to make sure that was an active call rather than an omission (PR descriptions are helpful sometimes ;-) )

Reply:
Yeah, my bad 😬 It was 2am in the morning and I just wanted to get the blasted tests to work...